### DISCUSSION TITLE
Training error in colab, " get_dataset(self)     525                 data = check_cls_dataset(self.args.data) --> 526             elif self.args.data.split(".")[-1] in {"yaml", "yml"} or self.args.task in {     527  "detect",  AttributeError: 'NoneType' object has no attribute 'split'"

### DISCUSSION THREAD

**Discussion by kwayne-08** [ultralytics/ultralytics#15962] [Docs]
I am trying to train an object detection dataset in Ultralytics using the colab option for training. But I am getting this error which suggests it can't find the dataset:

 get_dataset(self)
    525                 data = check_cls_dataset(self.args.data)
--> 526             elif self.args.data.split(".")[-1] in {"yaml", "yml"} or self.args.task in {
    527                 "detect",

AttributeError: 'NoneType' object has no attribute 'split'

How can I correct this?

**Comment by glenn-jocher** [comment]
@kwayne-08 it looks like the error is due to `self.args.data` being `None`. Please ensure that the path to your dataset YAML file is correctly specified when calling the `model.train()` function. If the issue persists, try updating to the latest version of the Ultralytics package.
